### DISCUSSION TITLE
How to know the api version ?

### DISCUSSION THREAD

**Discussion by MJx9977** [EndstoneMC/endstone#89] [Q&A]
How to know which endstone api version is compatible with my sv to make a plugin ?

**Comment by ye111566** [comment]
Looking at the https://endstone.dev in order to ensure whether the api is available in the version you used.You can also use the command in cmd:"endstone --version" to check your endstone version.In ordinary circumstances,the api-version is writen in the plugin.The same first and second letter of version means between endstone and plugin means this plugin can be loaded when the endstone server is starting.
